### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by kdaily**
Uses same license as `synapser` and `synapsePythonClient`.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (karawoo) [inline LICENSE]: Need to put in the year and Sage as the copyright owner

**Comment by kdaily** [comment]
@karawoo change made!

**Comment by karawoo** [comment]
@kdaily it doesn't look like you updated the license?

**Comment by kdaily** [comment]
Oops, that was below the fold in my email!

**Comment by karawoo** [comment]
I don't know if it matters but I've usually seen it without the brackets: "Copyright 2018 Sage Bionetworks"

**Comment by kdaily** [comment]
and indeed, the instructions say 'Don't include the brackets!' #rtfm

**Pull request merged**